user_guides/jakartaee/src/main/jbake/content/config.adoc

```diff
@@ -1041,7 +1003,7 @@ ant populateMailbox
 
 [[GJKPO]][[jakarta-restful-web-services-test-setup]]
 
-5.4.9 Jakarta RESTful Web Services Test Setup
+5.4.8 Jakarta RESTful Web Services Test Setup
```

Comment:
Might need to correct the following 5.4.9.1 to 5.4.8.1 etc Can be done in a separate change too

Reply:
Thanks, that does trickle down below as well for the numbering change.

Reply:
+1, I just renumbered the other below sections until we reach 5.5 :-), I'll include these changes along with the assertion updates.  I suspect that the assertion updates + doc changes can involve a few rounds of discussion + further changes so want to do that separately.